lib/stat_tracker.rb

```diff
@@ -158,6 +158,33 @@ def total_team_wins(team_id, season)
     team_wins_as_home(team_id, season) + team_wins_as_away(team_id, season)
   end
 
+  def game_ids_by_season(season)
+    filter_by_season(season).map do |game|
+      game.game_id
+    end.sort
+  end
+
+  def team_tackles(season)
+    team_season_tackles = {}
+    games = @game_teams.find_all do |game|
+      game_ids_by_season(season).include?(game.game_id)
+    end
+    games.each do |game|
+      if team_season_tackles[game.team_id]
+        team_season_tackles[game.team_id] += game.tackles
+      else
+        team_season_tackles[game.team_id] = game.tackles
+      end
+    end
+    team_season_tackles
+  end
+
+  def team_identifier(team_id)
+    @teams.find do |team|
+      team.team_id == team_id
+    end.team_name
+  end
```

Comment:
This method already exists (team_names_by_team_id), however I would opt to keep yours as it is a little cleaner.